src/tests/test_config.py

```diff
@@ -104,3 +115,31 @@ def test_defaults(self):
         self.assertEqual(
             self.config.safeGetInt(
                 'bitmessagesettings', 'maxaddrperstreamsend'), 500)
+
+    def test_disable_identity(self):
```

Comment:
both two new methods should do 3 things:
- change
- change already changed
- changed non-existing

No need for `try`/`except`. There is `AssertRaises` for that. No need for `if` either.